src/App.css

```diff
@@ -439,20 +439,29 @@ label.hover {
 
 
 /* List Styling */
-.grocery-list label {
-  padding-right: 12px;
-}
-
-.grocery-list li {
+.check-list li {
   border-radius: 4px;
   margin: 12px;
 }
 
-.grocery-list input {
-    width: 18px;
+.checkbox-input input {
```

Comment:
I think calling a `div` something like `.checkbox-input` is a bit ambiguous. The `div` is a wrapper, if anything. You could just call this `.checkbox`.

Additionally, it's worth mentioning that using raw element selectors like `input` in this selector are less performant than using `id` or `class`. [Check out this interesting article](https://csswizardry.com/2011/09/writing-efficient-css-selectors/) to learn more about it.

I don't think you need to address anything here, but if you *wanted to* you could update it to be `.checkbox .checkbox-input`